integration_tests/mockApis/logging.ts

```diff
@@ -2,21 +2,12 @@ import { SuperAgentRequest } from 'superagent'
 import { stubForWorkload } from './wiremock'
 
 export default {
-  stubSendComparisionLogToWorkload: (notesChanged: boolean = true): SuperAgentRequest => {
+  stubSendComparisonLogToWorkload: (): SuperAgentRequest => {
     return stubForWorkload({
       request: {
         method: 'POST',
         urlPattern: `/allocations/contact/logging`,
-        bodyPatterns: [
-          {
-            equalToJson: JSON.stringify({
-              crn: 'J678910',
-              teamCode: 'TM2',
-              isSPOOversightAccessed: true,
-              notesChanged,
-            }),
-          },
-        ],
+        bodyPatterns: [],
```

Comment:
Not a complaint, but I'm wondering why this works with no body pattern to match. Is it stubbing when there's no payload or when theres no body pattern, does it just match everything sent to that endpoint without worrying what's in the body? If it's the latter, that would be good to know

Reply:
Yes that appears to be the case. Discovered by accident when I forgot to put the body into another stub I added.  So now I use this stub for both cases